app/helper/WebsiteVersionPublisher.java

```diff
@@ -0,0 +1,428 @@
+/*
+ * Copyright 2019 hbz NRW (http://www.hbz-nrw.de/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package helper;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import actions.Modify;
+import actions.Modify.UpdateNodeException;
+import archive.fedora.CopyUtils;
+import models.Gatherconf;
+import models.Gatherconf.CrawlerSelection;
+import models.Node;
+import play.Logger;
+import play.Play;
+import com.yourmediashelf.fedora.client.request.GetDatastream;
+import com.yourmediashelf.fedora.client.request.GetDatastreamDissemination;
+import com.yourmediashelf.fedora.client.response.FedoraResponse;
+import com.yourmediashelf.fedora.client.response.GetDatastreamResponse;
+
+/**
+ * Methoden zum Veröffentlichen von Websites (openWayback AccessPoint
+ * "weltweit"). Methoden zum Zurückziehen veröffentlichter Websites (openWayback
+ * AccessPoint "lesesaal" oder "wayback").
+ * 
+ * @author I. Kuss, hbz
+ */
+public class WebsiteVersionPublisher {
+
+	private static final Logger.ALogger WebgatherLogger =
+			Logger.of("webgatherer");
+	// Das Unterverzeichnis in public-data/ (volle Pfadangabe) als Java-Klasse
+	// "File"
+	private static File publicCrawlDir = null;
+
+	/**
+	 * Veröffentlicht eine Webpage-Version (=Webschnitt), indem es sie in der
+	 * Openwayback-Kollektion "weltweit" anlegt.
+	 * 
+	 * @param node der Knoten des Webschnittes
+	 * @throws RuntimeException Ausnahme aufgetreten
+	 */
+	public static void publishWebpageVersion(Node node) throws RuntimeException {
+		WebgatherLogger.info("Jetzt wird ein Webschnitt veröffentlicht.");
+		try {
+			getConfFromFedora(node.getPid(), node);
+			Gatherconf conf = Gatherconf.create(node.getConf());
+			WebgatherLogger.debug("conf=" + conf.toString());
+			createSoftlinkInPublicData(node, conf);
+			setOpenwaybackLinkToPublicAccessPoint(node, conf);
+		} catch (Exception e) {
+			WebgatherLogger.error("Webpage Version " + node.getPid()
```

Comment:
kommt die Error message jetzt nicht doppelt? Weil in Resource.java#patchResource auch noch einmal geloggt wird.